README.md

```diff
@@ -20,13 +20,13 @@
 
 	`ve` is the name of the virtual environment
 
-3. Activate the virtual environment
+3. Activate the virtual environment (Should always work in virtual Env.)
 
 	```
 	source ve/bin/activate
 	```
 
-4. Install Django
+4. Install Django latest version
 
 	```
 	pip install django
```

Comment:
The following changes would be better:
* `Install Django latest version` :arrow_right: `Install the requirements`
* `pip install django` :arrow_right: `pip install -r requirements.txt`

Reply:
I meant that you change "Install Django latest version" to "Install the requirements" and "pip install django" to "pip install -r requirements.txt"

Reply:
😸 oopsie